turtle.js

```diff
@@ -490,12 +474,10 @@ const _main = function() {
     }
 
     // Execute the program in the command box when the user presses "Run" button or any "Enter" key
-    doc.getElementById('runButton').addEventListener('click', runCommand);
-    cmdBox.addEventListener('keydown', function(e) {
-        if (e.key == "Enter") runCommand();
-    });
+    _doc.getElementById('runButton').addEventListener('click', runCommand);
+    cmdBox.addEventListener('keydown', e => { if (e.key == "Enter") runCommand() });
```

Comment:
Another `if ()` elimination:
`cmdBox.addEventListener('keydown', e => e.key == "Enter" && runCommand());`

Reply:
As said previously, I'm waiting for @bjpop opinion